js/script.js

```diff
@@ -0,0 +1,106 @@
+
+//creates an array of 9 elements based on the amount of divs on the page
+var boardSpace = document.getElementsByClassName("boardSpace");
+
+//starts off with playing X starting the game
+var whichPlayer = 'x';
+var isWinner = false;
+
+//listens for click on X div to close alert windows
+document.getElementById('exit').addEventListener("click", function(){
+	document.getElementById('alert').style.display = 'none';
+})
+
+//creates arrays for which spaces have been selected for X and O
+var selectedArrayX = [];
+var selectedArrayO = [];
+
+//populates the selected array to empty values so they can be assigned
+for(var i = 0; i < boardSpace.length; i++){
+	selectedArrayX[i] = "";
+	selectedArrayO[i] = "";
+}
+
+//reset function used in new game button to reset game
+var reset = function(){
+	for(var i = 0; i < boardSpace.length; i++){
+			selectedArrayX[i] = "";
+			selectedArrayO[i] = "";
+	}
+	document.getElementById('winner').innerHTML= "<div id ='exit' class='x'></div>";
+	whichPlayer = 'x';
+}
+
+//array of winning arrays
+var winningArrays = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [2, 4, 6], [0, 4, 8]];
+
+//matches winning arrays with selected arrays to decide if anyone won the game
+var getWinner = function(boardArray){
+	for(var i = 0; i < winningArrays.length; i++){
+			var winningA = winningArrays[i][0];
```

Comment:
I like this method a lot for finding the winning solution. It's a lot more concise than having to check manually with a loop. Only exception is if you want to expand the board size to 4x4, or larger.